### PR TITLE
Package signing

### PR DESCRIPTION
**Pull request by TravisEz13**
Add script to generate package signing XML

### CODE REVIEW

File: tools/releaseBuild/generatePackgeSigning.ps1 [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        [Parameter(Mandatory)]
+        [System.Xml.XmlDocument]$XmlDoc,
+        [Parameter(Mandatory)]
+        [System.Xml.XmlElement]$Job
+    )
+
+    if(Test-Path -Path $file)
+    {
+        $name = Split-Path -Leaf -Path $File
+        $null = $fileElement = $XmlDoc.CreateElement("file")
```
Comment (daxian-dbw) [inline tools/releaseBuild/generatePackgeSigning.ps1]: is the `$null =` part still necessary?

**Review by daxian-dbw — APPROVED** [review]
With a minor comment

**Comment by daxian-dbw** [comment]
@TravisEz13 Feel free to merge it.

File: tools/releaseBuild/generatePackgeSigning.ps1 (hunk 1 above)
Comment (TravisEz13) [reply tools/releaseBuild/generatePackgeSigning.ps1]: nope...  Will mered and fix later.

**Pull request merged**